docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -57,6 +57,26 @@ Such arguments could include passwords or other sensitive information.
 You should also make sure no stack trace is ever visible to end users of production sites,
 though visible arguments are unsafe even if the stack traces only show up in log files.
 
+### Disable error output from PHP
+
+Symfony in production mode prevents exception messages from being visible to end users.
+However, if Symfony fails to boot properly, such exceptions may end up being visible, including stack traces.
+This can be prevented by [disabling error message output in PHP](https://www.php.net/manual/en/language.errors.basics.php).
+These `php.ini` configuration values should be used on production sites.
+When using '[[= product_name_cloud =]]' / Platform.sh the same settings can be configured in '[[= product_name_base =]]' DXP's `.platform.app.yaml` file.
```

Comment:
```suggestion
When using [[= product_name_cloud =]] the same settings can be configured in [[= product_name_base =]] DXP's `.platform.app.yaml` file.
```

Preview link: https://ez-systems-developer-documentation--2423.com.readthedocs.build/en/2423/infrastructure_and_maintenance/security/security_checklist/#symfony-production-mode
<img width="933" alt="Zrzut ekranu 2024-07-3 o 13 23 07" src="https://github.com/ibexa/documentation-developer/assets/10993858/9400186f-c8de-477e-9c4f-f01c8fcd7cc3">

No need to quote the variables, the end result does not look good.

I'm not sure if we should mention Platform.sh here - for us the product name is Ibexa Cloud and I'd focus on it

Reply:
Continuing on @mnocon sugggestion, I would factorize the product name, and add a comma:
```suggestion
When using [[= product_name_cloud =]], the same settings can be configured in [[= product_name =]]'s `.platform.app.yaml` file.
```